example/accountKey/AccountKeyRoleBased_01_accountUpdate.js

```diff
@@ -0,0 +1,85 @@
+const ethers = require("ethers");
+const { Wallet } = require("../../dist/src/ethers"); // require("@klaytn/sdk-ethers");
+const { RLP, HexStr } = require("../../dist/src/core/util")
+const fs = require('fs');
+
+// 
+// AccountKeyRoleBased Step 01 - account update
+// https://docs.klaytn.foundation/content/klaytn/design/accounts#accountkeyrolebased 
+//
+//   gasLimit: Must be large enough 
+// 
+//   create a new account for testing 
+//   https://baobab.wallet.klaytn.foundation/ 
+//
+
+const sender_priv = '0x3656ff020a8bdb90c991f571eb5615b4ee5d675a729b0f48c756fa980b0da71e' 
+const sender = '0x9b4284806060423079e612203c22e8cb48b9870e' 
```

Comment:
Please load keys in a consistent manner.
The key loading part takes up a significant portion of example codes. Not by line numbers, but by the mental cost.

Either
1. hardcode string in example code
	```js
	sender_priv = '0x3656ff020a8bdb90c991f571eb5615b4ee5d675a729b0f48c756fa980b0da71e';
	sender_addr = '0x9b4284806060423079e612203c22e8cb48b9870e';
	pub1 = '0x04aaaa...';
	pub2 = '0x04bbbb...';
	pub3 = '0x04cccc...';
	```
2. readfile into global
	```js
	sender_priv = fs.readFileSync('./example/key0.priv', 'utf8');
	sender_addr = ethers.computeAddress(sender_priv);
	pub1 = ethers.computeAddress(fs.readFileSync('./example/key2.pub', 'utf8');
	```
3. file reading function
	```
	function readPub(path) { ... }
	function readPriv(path) { ... }
	sender_priv = readPriv('./example/key0.priv');
	pub1 = readPub('./example/key2.pub');
	```

Reply:
All example has to be checked
It will be updated later.